package-lock.json

```diff
@@ -14122,6 +14127,14 @@
         }
       }
     },
+    "react-visibility-sensor": {
```

Comment:
I did not find usage of 
`react-visibility-sensor
`and
`classnames`
Do we plan to add them in the future? 
It's not important for approving, just for information